tests/unit/test_enums.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_compression_enums():
```

Comment:
I wonder how useful this test is? Seems an awful lot like a [change-detector test](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) to me. I'd be fine adding the constant without adding the test.

Alternatively, maybe there's a system test we could write to make sure this is synced with the [bigquery discovery document](https://bigquery.googleapis.com/discovery/v1/apis/bigquery/v2/rest)? But even then, compression isn't a true enum. The allowed values are only listed in the documentation string from what I can tell.

Reply:
@tswast  

I will remove this test.

In terms of attempting to ensure that this matches the discovery docs... the terms referenced by the docs are present in the description, as you note, so we would need some mechanism to extract them from the discovery doc, which feels somewhat fragile (ie extract all words that are ALL CAPS and deduplicate them). Thoughts?

```
"JobConfigurationExtract": {
      ...
      "properties": {
        "compression": {
          "description": "Optional. The compression type to use for exported files.
Possible values include DEFLATE, GZIP, NONE, SNAPPY, and ZSTD. The
default value is NONE. Not all compression formats are support for all
file formats. DEFLATE is only supported for Avro. ZSTD is only supported
for Parquet. Not applicable when extracting models.",
          "type": "string"
        },
```

Reply:
Agreed. Without a structured representation of the allowed values, it's too fragile.